doc/INSTALL.md

```diff
@@ -23,6 +23,16 @@ For the following Linux distributions and package managers 0.8 packages are avai
 For [OpenSUSE, CentOS, Fedora, RHEL and SLE](https://build.opensuse.org/package/show/home:bekun:devel/elektra)
 Kai-Uwe Behrmann kindly provides packages [for download](http://software.opensuse.org/download.html?project=home%3Abekun%3Adevel&package=libelektra4).
 
+### Ubuntu-Bionic
+
+To use the Ubuntu-Bionic packages following steps need to be made:
+
+1. Run `sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D919CE8B27A64C16656FCA9FF1532673651F9C6C` to obtain the key.
+
+2. Add `deb [trusted=yes] https://ubuntu-bionic-repo.libelektra.org/ bionic main` into `/etc/apt/sources.list`
```

Comment:
`[trusted=yes]` is only needed if the key was not imported.

Reply:
Thank you.